src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Copper/CorruptedFlaminca.cs

```diff
@@ -7,21 +7,26 @@ namespace Cynthia.Card
 {
     [CardEffectId("70013")]//堕落的佛兰明妮卡
     public class CorruptedFlaminca : CardEffect
-    {//弃掉牌库里一张倾盆大雨，然后在敌我双方同排降下倾盆大雨。
+    {// Discard a rain from your deck and spawn rain on its row an the opposite, if there is no rain in the deck, apply rain on both sides and damage self by 4
         public CorruptedFlaminca(GameCard card) : base(card) { }
         public override async Task<int> CardPlayEffect(bool isSpying, bool isReveal)
         {
-            var list = Game.PlayersDeck[Card.PlayerIndex].Where(x => x.Status.CardId == CardId.TorrentialRain).Mess(Game.RNG);
-            if (list.Count() == 0) { return 0; }
 
-            var result = await Game.GetSelectMenuCards(Card.PlayerIndex, list.ToList(), 1, "选择丢弃一张牌");
-
-            if (result.Count() == 0) { return 0; }
-
-            await result.First().Effect.Discard(Card);
+        var list = Game.PlayersDeck[Card.PlayerIndex].Where(x => x.Status.CardId == CardId.TorrentialRain).Mess(Game.RNG);
+        if (list.Count() == 0) // if there is no rain in the deck
+            {
```

Comment:
nitpicking: the indent here is incorrect. It should be
```
if (...)
{

}
```

I believe auto formatter in VSCode or VS can help you format those.

Reply:
Thanks for telling me, those are bad habits from other languages, will correct.